lib/modules/manager/custom/jsonata/readme.md

```diff
@@ -0,0 +1,182 @@
+
+- `fileMatch` works the same as any manager
+- `matchStrings` is a `JSONata` custom manager concept and is used for configuring a jsonata queries
+
+#### Information that Renovate needs about the dependency
+
+Before Renovate can look up a dependency and decide about updates, it must have this info about each dependency:
+
+| Info type                                            | Required | Notes                                                     | Docs                                                                           |
+| :--------------------------------------------------- | :------- | :-------------------------------------------------------- | :----------------------------------------------------------------------------- |
+| Name of the dependency                               | Yes      |                                                           |                                                                                |
+| `datasource`                                         | Yes      | Example datasources: npm, Docker, GitHub tags, and so on. | [Supported datasources](../../datasource/index.md#supported-datasources)       |
+| Version scheme to use. Defaults to `semver-coerced`. | Yes      | You may set another version scheme, like `pep440`.        | [Supported versioning schemes](../../versioning/index.md#supported-versioning) |
+
+#### Required fields to be present in the resulting structure returned by the jsonata query
+
+You must:
+
+- Capture the `currentValue` of the dependency
+- Capture the `depName` or `packageName`. Or use a template field: `depNameTemplate` and `packageNameTemplate`
+- Capture the `datasource`, or a use `datasourceTemplate` config field
+
+#### Optional fields you can include in the resulting structure
+
+You may use any of these items:
+
+- `depType`, or a use `depTypeTemplate` config field
+- `versioning`, or a use `versioningTemplate` config field. If neither are present, Renovate defaults to `semver-coerced`
+- `extractVersion`, or use an `extractVersionTemplate` config field
+- `currentDigest`
+- `registryUrl`, or a use `registryUrlTemplate` config field. If it's a valid URL, it will be converted to the `registryUrls` field as a single-length array
+- `indentation`. It must be either empty, or whitespace only (otherwise `indentation` will be reset to an empty string)
+
+### Usage
+
+To configure it, use the following syntax:
+
+```javascript
+{
+  "customManagers": [
+    {
+      "customType": "jsonata",
+      "fileFormat": "json",
+      "fileMatch": ["<file match pattern>"],
+      "matchStrings": ['<query>'],
+      ...
+    }
+  ]
+}
+```
+
+Where `<query>` is a [JSONata](https://docs.jsonata.org/overview.html) query that transform the contents into a JSON object with the following schema:
+
+To be effective with the JSONata manager, you should understand jsonata queries. But enough examples may compensate for lack of experience.
```

Comment:
```suggestion
Creating your Renovate JSONata manager config is easier if you understand JSONata queries.
We recommend you follow these steps:

1. Read the official JSONata query language docs
2. Check our example queries below
3. You're ready to make your own config

Alternatively you can "try and error" to a working config, by adjusting our examples.
```

How about something like this? I think we want to nudge newcomers to read the JSONata docs, then read our examples, and only then make their own config.

I expect most users prefer to "try and error their way into a working config" instead of reading the docs.